packages/skin/src/routes/_index/component/card/+page.marko

```diff
@@ -0,0 +1,566 @@
+        </div>
+        <div class="card__main">
+            <div class="card__description">
+                Hockey cards collected by enthusiasts who treat them like fine wine – stored in temperature-controlled cellars and discussed in hushed, reverent tones.
+            </div>
+        </div>
+    </a>
+</highlight-code>
+
+    <h2 id="card-horizontal">Horizontal Card</h2>
+
+    <p>
+        Card also has a layout variant that can displayed it in a horizontal layout. This is useful when you want to display an image alongside the content. To achieve this, use the <highlight>.card--horizontal</highlight> modifier.
+    </p>
+
+<demo>
+    <span class="card card--horizontal card--outlined">
+        <div class="card__media">
+            <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+        </div>
+        <div class="card__main">
+            <div class="card__overline">
+                <svg class="icon icon--16" height="16" width="16">
+                    <use href="#icon-authenticity-guarantee-filled-16-colored" />
+                </svg>
+                <span class="card__overline-text">Authenticity Guaranteed</span>
+            </div>
+            <h3 class="card__title">Authentic Rookie Cards</h3>
+            <div class="card__description">
+                Museum-quality rookie cards from three sports that prove some people's side hustles become billion-dollar careers while others struggle to keep houseplants alive.
+            </div>
+            <div class="card__footer">
+                <button class="btn btn--tertiary">See Details</button>
+            </div>
+        </div>
+    </span>
+</demo>
+
+<highlight-code type="html">
+<span class="card card--horizontal card--outlined">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <div class="card__overline">
+            <svg class="icon icon--16" height="16" width="16">
+                <use href="#icon-authenticity-guarantee-filled-16-colored" />
+            </svg>
+            <span class="card__overline-text">Authenticity Guaranteed</span>
+        </div>
+        <h3 class="card__title">Authentic Rookie Cards</h3>
+        <div class="card__description">
+            Museum-quality rookie cards from three sports that prove some people's side hustles become billion-dollar careers while others struggle to keep houseplants alive.
+        </div>
+        <div class="card__footer">
+            <button class="btn btn--tertiary">See Details</button>
+        </div>
+    </div>
+</span>
+</highlight-code>
+
+    <h2 id="card-self-actionable">Self-Actionable Card</h2>
+
+    <p>
+        The self-actionable card is a variant that allows the entire card to be clickable, leading to a specific action or page. A Self-actionable card <strong>cannot</strong> contain interactive elements such as buttons or links within it.
```

Comment:
Self actionable cards also cannot contain structured data, like nested headings, lists, etc. because screen readers will ignore all of that when reading out button/link text. It's worth noting that here